Makefile

```diff
@@ -1,80 +1,95 @@
-
-# Image URL to use all building/pushing image targets
-IMG ?= cloud-credential-operator:latest
-
-DISTRO ?= $(shell if which lsb_release &> /dev/null; then lsb_release -si; else echo "Unknown"; fi)
-
-# Default fedora to not using sudo since it's not needed
-ifeq ($(DISTRO),Fedora)
-	SUDO_CMD =
-else # Other distros like RHEL 7 and CentOS 7 currently need sudo.
-	SUDO_CMD = sudo
-endif
-
-all: test manager
-
-# Run tests
-test: generate fmt vet manifests
-	go test ./pkg/... ./cmd/... -coverprofile cover.out
-
-# Run tests without attempting any code generation. (CI)
-test-no-gen: fmt vet
-	go test ./pkg/... ./cmd/... -coverprofile cover.out
-
-.PHONY: test-sec
-test-sec:
-	@which gosec 2> /dev/null >&1 || { echo "gosec must be installed to lint code";  exit 1; }
-	gosec -severity medium --confidence medium -quiet ./...
-
-# Build manager binary
-manager: generate fmt vet
-	go build -o bin/manager github.com/openshift/cloud-credential-operator/cmd/manager
-
-# Build without attempting any code generation. (CI)
-build-no-gen: fmt vet
-	go build -o bin/manager github.com/openshift/cloud-credential-operator/cmd/manager
-
-# Run against the configured Kubernetes cluster in ~/.kube/config
-run: generate fmt vet
-	go run ./cmd/manager --log-level=debug
```

Comment:
Make install and run are both used for devel and documented in README, we need to either preserve these or update the docs for how to do these standard dev tasks.

Reply:
i can add those targets back